### DISCUSSION TITLE
Stable Diffusion on a 4600G!

### DISCUSSION THREAD

**Discussion by SalmonPete** [ChefKissInc/NootedRed#164] [Show and tell]
Wow!, amazing work NootedRed!. With your latest kext Stable Diffusion finally works, took just a couple of minutes!.

![Screenshot 2023-09-07 at 10 00 54 AM](https://github.com/ChefKissInc/NootedRed/assets/143473778/16a6c103-b7b1-4262-b298-9a92f1e6b4f1)

**Comment by Kanepan** [comment]
Well done!
